db/migrate/20200402152601_add_moderator_id_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddModeratorIdToComments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :comments, :moderator_id, :bigint, references: "users"
```

Comment:
I think `big_int` and `references: "users"` is the right choice but not entirely sure here.